configs/11.0/build.mk

```diff
@@ -1 +0,0 @@
-../10.0/build.mk
```

Comment:
There are two changes involved:
- pkg_build_deb.sh.  Before the change, this set __AT_VER_REV_INTERNAL__ from "at_ver_rev_internal", but without any dots.  This script is not release-dependent.  After the change, it sets __AT_VER_REV_INTERNAL__ directly from "at_ver_rev_internal", including the dots; and it sets __AT_VER_REV_INTERNAL_NODOT__ from "at_ver_rev_internal" without the dots.  Since the script is not release-dependent, all release-dependent files which used __AT_VER_REV_INTERNAL__ needed to change.  Thus the following additional changes.

- ./configs/7.1/deb/monolithic/rules
- ./configs/9.0/deb/monolithic/rules
- ./configs/10.0/deb/monolithic/rules
  all of these simply had changes to use the new variable:
--
-             > debian/tmp/etc/cron.d/__AT_VER_REV_INTERNAL___ldconfig
+             > debian/tmp/etc/cron.d/__AT_VER_REV_INTERNAL_NODOT___ldconfig
--

So, this change should work in all releases.